### PR TITLE
[16.0][OU-ADD] product

### PR DESCRIPTION
**Pull request by remytms**
Renamed field name -> partner_id from model product.supplierinfo

### CODE REVIEW

File: openupgrade_scripts/scripts/product/16.0.1.2/upgrade_analysis_work.txt [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---Models in module 'product'---
+new model product.tag
```
Comment (legalsylvain) [inline openupgrade_scripts/scripts/product/16.0.1.2/upgrade_analysis_work.txt:2]: It seems that odoo integrates now oca features présent in https://github.com/OCA/product-attribute/tree/14.0/product_template_tags

If yes some migrations scripts could be added in a second time. (Not in the scope of this PR)
Cc @ivantodorovich 
@patrickrwilson

**Review by legalsylvain — APPROVED** [review]
Thanks ! Code review. Lgtm.

**Comment by remytms** [comment]
@legalsylvain Can you reference this PR with `/ocabot migration product` ? :)

**Review by SonCrits — APPROVED** [review]

**Pull request merged**